### ISSUE TITLE
Allow Removing Projects

### ISSUE THREAD

**Issue by AliWaheed** [midas-research/audino#52]
Great work team, really enjoyed audino, using it first hand tells you how it has been geared towards production, especially the data uploading logic is just amazing. Really love what you guys have done I feel just a few features and this would be a really amazing product.

I have seen similar issues requesting CRUD operations for users and data, I would like to add to allow it for projects too. Thank you for your hard work !

**Comment by manrajgrover** [comment]
Yes! We're working on adding complete CRUD for all entities. Will update here once release is made.
